File: stepik-java-api/src/main/java/org/stepik/api/actions/StepikUsersAction.java
package org.stepik.api.actions;

import org.jetbrains.annotations.NotNull;
import org.stepik.api.client.StepikApiClient;
import org.stepik.api.queries.users.StepikUsersGetQuery;

/**
 * @author meanmail
 */
public class StepikUsersAction extends StepikAbstractAction {
    public StepikUsersAction(@NotNull StepikApiClient stepikApiClient) {
        super(stepikApiClient);
    }

    @NotNull
    public StepikUsersGetQuery get() {
        return new StepikUsersGetQuery(this);
    }
}

File: stepik-java-api/src/main/kotlin/org/stepik/api/actions/StepikUsersAction.kt
package org.stepik.api.actions

import org.stepik.api.client.StepikApiClient
import org.stepik.api.queries.users.StepikUsersGetQuery

class StepikUsersAction(stepikApiClient: StepikApiClient) : StepikAbstractAction(stepikApiClient) {
    
    fun get(): StepikUsersGetQuery {
        return StepikUsersGetQuery(this)
    }
}
